config/csm/rest_response_data.yaml

```diff
@@ -62,6 +63,26 @@
     - "Invalid request body: {'role': ['Must be one of: admin, manage, monitor.']}"
     - "Invalid request body: {'role': ['Must be one of: admin, manage, monitor.'], 'password': [\"Password Policy Not Met. Must be more than 8 characters. Must contain at least one Uppercase Alphabet. Must contain at least one Lowercase Alphabet. Must include !@#$%^&*()_+-=[]{}|'.\"]}"
     - "Invalid request body: {'password': [\"Password Policy Not Met. Must be more than 8 characters. Must contain at least one Uppercase Alphabet. Must contain at least one Lowercase Alphabet. Must include !@#$%^&*()_+-=[]{}|'.\"], 'role': ['Must be one of: admin, manage, monitor.']}"
+    - "Invalid request body: {'password': ['Password Policy Not Met. Must contain at least one Numeric value.']}"
+    - "{'sortby': ['Must be one of: user_id, username, email, user_type, role, created_time, updated_time, email_address, user_role.']}"
+    - "{'dir': ['Must be one of: desc, asc.']}"
+    - "username cannot be modified"
+    - "Insufficient information in request body"
+    - "_schema: Empty value for access_key"
+    - "_schema: Empty value for secret_key"
+    - "$A: Field may not be null."
+    - "$A: Must be one of: s3."
+    - "$A: Field may not be null."
+    - "$A: not a valid integer."
+    - "$A: field may not be null."
+    - "$A: not a valid boolean."
+    - "$A: field may not be null."
```

Comment:
we have "Field may not be null" already, you added 4 new, we can have only 1

Reply:
Ok Sure, I will update this one.

Reply:
Updated the response data file and impacted test cases mapping. cc @RajeshPratapSInghChouhan